src/main/java/invtweaks/InvTweaksHandlerShortcuts.java

```diff
@@ -437,16 +437,18 @@ private void moveAll(ShortcutConfig shortcut, ItemStack stackToMatch) {
 
                     // Continue if movement succeeded, there is another slot to try, or we're dropping items.
                     // In reverse: fail if movement failed, AND there are no other slots AND we're not dropping.
-                    if(success || (newIndex != toIndex) || (shortcut.action == ShortcutSpecification.Action.DROP)) {
+                    if (newIndex == toIndex) {
```

Comment:
For multi-drops, I think the condition to break needs to be `newIndex == toIndex && shortcut.action != ShortcutSpecification.Action.DROP`

The indices will always be the same on a drop action, but that's intentional because they specify a 'magic' slot that doesn't really exist.

Then the second condition can just be `else if(success)`